modules/DMZ/utils/config_options.py

```diff
@@ -0,0 +1,27 @@
+class ConfigOptions(object):
+
+    #Initialize the class
+    def __init__(self):
+        self.training_file_name = None
+        self.test_file_name = None
+        self.separator_type = None
+        self.prediction_type = None
+        self.target_column = None
+        self.get_config_options()
+        self.validate_options()
+
+    #Get input from user
```

Comment:
I'm happy to hear your ideas about it if you have something you think might be better.